File: boj/Implemenation/print/23811/23811.cpp
#include <bits/stdc++.h>
using namespace std;
int n;

void printLong(){
  for(int i = 0; i < n; i++) {
    for(int j = 0; j < n * 5; j++){
      cout << "@";
    }
    cout << '\n';
  }
}

void printShort(){
  for(int i = 0; i < n; i++){
    for(int j = 0; j < n; j++){
      cout << "@";
    }
    cout << '\n';
  }
}

int main(){
  cin >> n;
  printLong();
  printShort();
  printLong();
  printShort();
  printLong();
}

File: boj/Implemenation/print/23811/23811.rs
use std::io;

pub fn print_long(n: i64){
  for _ in 0..n {
    for _ in 0..n*5{
      print!("@");
    }
    println!();
  }
}

pub fn print_short(n: i64){
  for _ in 0..n {
    for _ in 0..n {
      print!("@");
    }
    println!();
  }
}

fn main(){
  let mut line = String::new();
  io::stdin().read_line(&mut line).expect("wrong io");
  let n = line.trim().parse::<i64>().unwrap();
  print_long(n);
  print_short(n);
  print_long(n);
  print_short(n);
  print_long(n);
}